README.md

```diff
@@ -1,10 +1,46 @@
-# CapsulaJS: Environment Registry
+### CapsulaJS: Environment Registry
 
-Capsula-hub will allow to load different versions of a project/service.
+This service allows to register and load different versions of a
+project/service environment.
 
-The aim of this service is to provide a way to register those into an environment
-and to allow to access them.
+An environment is an object that contains an `envKey` and an `env`
+(which is an array of `accessPoints`)
 
-This service will be able to be called by a CI-CD pipeline in order to keep at 
-disposal each version of projects/services automatically after pushing 
-modifications.
\ No newline at end of file
+The service exposes two methods: `register` and `environments$`
+
```

Comment:
Is envRegistry should exist in global namespace? Or there is some way to get envRegistry instance?

Reply:
@vvpyshnenko 
Actually, it needs to be imported and instantiated (with a token) to be able to use the methods. In the future, it will be possible to call it as a CLI tool.

Reply:
Thats why I meant to put in example appropriate imports and instantiation